lib/ueberauth/strategy/linkedin.ex

```diff
@@ -41,6 +36,7 @@ defmodule Ueberauth.Strategy.LinkedIn do
     if token.access_token == nil do
```

Comment:
I've been told by Dialyzer that this will never return `true`. What do you think of this result from Dialyzer?